renovate.json

```diff
@@ -0,0 +1,48 @@
+{
+  "extends": [
+    "config:base"
+  ],
+  "packageRules": [
+    {
+      "automerge": true,
+      "matchUpdateTypes": [
+        "patch",
+        "pin",
+        "digest",
+        "minor"
+      ]
+    },
+    {
+      "description": "Tag the waddlers Github Team for major updates",
+      "matchUpdateTypes": [
+        "major"
+      ],
+      "reviewers": [
+        "team:waddlers"
+      ]
+    },
+    {
+      "automerge": true,
+      "description": "Group minor and patch updates into a single PR",
+      "groupName": "dependencies",
+      "managers": [
+        "terraform",
+        "pre-commit",
+        "gomod",
+        "dockerfile"
+      ],
+      "matchUpdateTypes": [
+        "minor",
+        "patch"
+      ]
```

Comment:
Nit: What about pin and digest?

Reply:
Bad copy/paste. Pushed a fix. Let me know.